vtex/faststore
# feat(gatsby-theme-store): Move auth pages to stores

## What's the purpose of this pull request?
This PR moves the login/account pages implementation to the stores. 

## How to test it?
This PR is only moving files from one place to the other, so it should not have any visible change. To test it just go to the login page and make sure nothing has changed

https://github.com/vtex-sites/btglobal.store/pull/684
https://github.com/vtex-sites/marinbrasil.store/pull/546
https://github.com/vtex-sites/storecomponents.store/pull/1015
https://github.com/vtex-sites/carrefourbrfood.store/pull/879
